### PR TITLE
MFTF: Use AdminLoginActionGroup for AdminLoginTest - easiest use of ActionGroup

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
I found it annoying that the easiest example of ActionGroup usage is incorrectly implemented. Decided to use AdminLoginActionGroup to AdminLoginTest case.

What is more - for me successful Admin Login should end up with successful logout, so that I added logout verification.

### Fixed Issues (if relevant)
Irrelevant

### Manual testing scenarios (*)
1. Run MFTF on AdminLoginTest

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by dmytro-ch** [comment]
Hi @lbajsarowicz,

the action group "logout" is too generic and it's being used widely.
In my opinion, it should not contain assertions.

CC @okolesnyk @rogyar

**Comment by lbajsarowicz** [comment]
@dmytro-ch Rebased to remove redundant change

File: app/code/Magento/Backend/Test/Mftf/Test/AdminLoginTest.xml [hunk 1]
```diff
@@ -20,11 +20,9 @@
             <group value="login"/>
         </annotations>
 
-        <amOnPage url="{{AdminLoginPage.url}}" stepKey="amOnAdminLoginPage"/>
-        <fillField selector="{{AdminLoginFormSection.username}}" userInput="{{_ENV.MAGENTO_ADMIN_USERNAME}}" stepKey="fillUsername"/>
-        <fillField selector="{{AdminLoginFormSection.password}}" userInput="{{_ENV.MAGENTO_ADMIN_PASSWORD}}" stepKey="fillPassword"/>
-        <click selector="{{AdminLoginFormSection.signIn}}" stepKey="clickOnSignIn"/>
+        <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
         <closeAdminNotification stepKey="closeAdminNotification"/>
```
Comment (dmytro-ch) [inline app/code/Magento/Backend/Test/Mftf/Test/AdminLoginTest.xml]: The `closeAdminNotification` is performed twice, isn't it?
https://github.com/magento/magento2/blob/bb3cedebb313a4f4cfaa488ab993c1ea36a7c75f/app/code/Magento/Backend/Test/Mftf/ActionGroup/LoginAsAdminActionGroup.xml#L19

**Review by dmytro-ch — COMMENTED** [review]
@lbajsarowicz, thank you for the update!
Could you please also check the minor issue mentioned CR?

File: app/code/Magento/Backend/Test/Mftf/Test/AdminLoginTest.xml (hunk 1 above)
Comment (lbajsarowicz) [reply app/code/Magento/Backend/Test/Mftf/Test/AdminLoginTest.xml]: Good point.

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-5271](https://jira.corp.magento.com/browse/ENGCOM-5271) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**